client/src/components/uis/MessageListItem.tsx

```diff
@@ -163,6 +171,10 @@ const decorateDate = (createdAt: string): string => {
   return date.fromNow();
 };
 
+const handleUrlPress = (url: string): void => {
+  Linking.openURL(url);
```

Comment:
I was not sure if this would work on web, but it does 👍

Reply:
I checked this works on web(in Chrome)!